src/main/java/com/google/devtools/build/lib/skyframe/ToolchainUtil.java

```diff
@@ -63,6 +65,7 @@ static ToolchainContext createToolchainContext(
       Environment env,
       String targetDescription,
       Set<Label> requiredToolchains,
+      Set<Label> execConstraintLabels,
```

Comment:
I know the precedents aren't great but please document the new parameter (at least).